From: =?UTF-8?q?Picca=20Fr=C3=A9d=C3=A9ric-Emmanuel?= <picca@synchrotron-soleil.fr>
Date: Sat, 26 Mar 2011 14:24:26 +0100
Subject: feature fix compilation with tango 7.2.0

---
 setup.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/setup.py b/setup.py
index b7a2a0b..7b41b8e 100644
--- a/setup.py
+++ b/setup.py
@@ -135,7 +135,7 @@ def uniquify(seq):
 
 include_dirs = [
     os.path.abspath('src'),
-    os.path.join(TANGO_ROOT, 'include'),
+    os.path.join(TANGO_ROOT, 'include/tango'),
     os.path.join(OMNI_ROOT, 'include'),
     os.path.join(NUMPY_ROOT, 'include'),
 ]
-- 
