From: =?UTF-8?q?Picca=20Fr=C3=A9d=C3=A9ric-Emmanuel?= <picca@synchrotron-soleil.fr>
Date: Thu, 14 Apr 2011 19:27:37 +0200
Subject: install the documentation with the debian docs file

---
 setup.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/setup.py b/setup.py
index d636e82..cb5348d 100644
--- a/setup.py
+++ b/setup.py
@@ -418,7 +418,7 @@ class install(dftinstall):
         return sphinx is not None
     
     sub_commands = list(dftinstall.sub_commands)
-    sub_commands.append(('install_html', has_html))
+    #sub_commands.append(('install_html', has_html))
 
 
 cmdclass['install'] = install
-- 
