From: =?UTF-8?q?Picca=20Fr=C3=A9d=C3=A9ric-Emmanuel?= <picca@synchrotron-soleil.fr>
Date: Sat, 26 Mar 2011 14:25:04 +0100
Subject: feature fix the doc generation

---
 setup.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/setup.py b/setup.py
index 7b41b8e..d04c46b 100644
--- a/setup.py
+++ b/setup.py
@@ -291,7 +291,7 @@ class build_ext(dftbuild_ext):
             #self.compiler.compiler_so = " ".join(compiler_pars)
         dftbuild_ext.build_extensions(self)
 
-cmdclass = {'build_ext' : build_ext }
+cmdclass['build_ext'] = build_ext
 
 if sphinx:
     from sphinx.setup_command import BuildDoc
-- 
